state.json

```
@@ -242,7 +242,8 @@
                 "name": "pocket",
                 "random": true,
                 "commands": [
-                    "https://i.imgur.com/EgoNBWl.jpg comment=\"Wifi update success\" end=2018-11-24T00:00:00"
+                    "https://i.imgur.com/EgoNBWl.jpg comment=\"Wifi update success\" end=2018-11-24T00:00:00".
+                    "https://docs.google.com/presentation/d/1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY/edit?usp=sharing"
```

This won't quite work, because Corsica would try to iframe the site, which Google Slide's doesn't allow. The correct way to do this would be

```suggestion
                    "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\""
```

That instructs Corsica to run some custom code that embeds the presentation correctly.